Tasks/VsTest/vstest.ts

```diff
@@ -174,12 +174,20 @@ function updateResponseFile(argsArray: string[], responseFile: string): Q.Promis
     argsArray.forEach(function (arr, i) {
         argsArray[i] = utils.Helper.modifyVsTestConsoleArgsForResponseFile(arr);
     });
-    fs.appendFile(responseFile, os.EOL + argsArray.join(os.EOL), function (err) {
+
+    let otherConsoleOptions : string = '';
+    if (!utils.Helper.isNullEmptyOrUndefined(vstestConfig.otherConsoleOptions)) {
+        otherConsoleOptions = vstestConfig.otherConsoleOptions;
+    } 
+
+    const appendString = os.EOL + argsArray.join(os.EOL) + os.EOL + otherConsoleOptions;
```

Comment:
Can be like this
let appendString = os.EOL + argsArray.join(os.EOL);
if (!utils.Helper.isNullEmptyOrUndefined(vstestConfig.otherConsoleOptions)) {
 +        appendString = appendString + os.EOL + otherConsoleOptions;
 +    }